src/compiler/proposals-and-stabilization/custom-test-jobs.md

```diff
@@ -0,0 +1,248 @@
+# Adding ecosystem/integration test jobs/components to rust-lang/rust CI
+
+## Scope
+
+This policy is applicable to proposals for adding new ecosystem and integration test jobs/components
+that involve building and testing additional artifacts which may cause [rust-lang/rust] PR CI or
+Full CI (the "CI") to fail, or produce a failure message, that may impact other [rust-lang/rust]
+contributors who also use the [rust-lang/rust] CI.
+
+For example (non-exhaustive):
+
+- Ecosystem test jobs: Rust for Linux or Fuchsia.
+- Integration test components: GCC codegen backend or Cranelift codegen backend.
+
+## Background
+
+[rust-lang/rust] runs a small set of build/test jobs on PR CI (faster, less expensive/exhaustive),
+and runs a much larger set of build/test jobs on Full CI. PR CI usually takes around an hour, while
+Full CI usually takes around 3 hours. Having ecosystem and integration test jobs/components that:
+
+- may spuriously fail; or
+- may genuinely fail but it's not clear who should be consulted about the failure or who is
+  responsible for fixing the failure; or
+- otherwise do not have well documented test job maintainers and failure protocol;
+
+can introduce a lot of friction and frustration for other contributors who now will have to deal
+with the failure or understand why the test job failed in the first place, which may now be blocking
+their PR. Everyone utilizing the [rust-lang/rust] CI is expected to use it responsibly.
+
+To help with this, please follow the process described below if you would like to propose adding an
+ecosystem/integration test job/component to the [rust-lang/rust] CI.
+
+## Process for adding an ecosystem/integration test job/component to rust-lang/rust CI
+
+![Flowchart of the process for adding an ecosystem/integration test job/component to rust-lang/rust
+CI](./custom-test-jobs/test-job-flow.svg)
+
+1. Ask the Infrastructure Team on zulip
+  ([#t-infra](https://rust-lang.zulipchat.com/#narrow/channel/242791-t-infra)) to check if there
+  would be capacity for the proposed test job/component.
+1. Propose using a [Major Change Proposal
```

Comment:
I know that Markdown might render it correctly, but shouldn't ths be `1), 2), 3)` etc.? :)

Reply:
Yeah, it will render correctly, but I can also change it to 1) 2) 3) in source :D

Reply:
Fixed it to use 1. 2. 3. instead.